test/suites/osconfig/clusterid.robot

```diff
@@ -83,10 +102,79 @@ Get MicroShift Cluster ID From Namespace
     Should Not Be Empty    ${clusterid}
     RETURN    ${clusterid}
 
+Extract Sos Report
+    [Documentation]    Extract Sos Report from the tar file
+    [Arguments]    ${sos_report_tarfile}
+
+    ${sos_report_dir}    ${rc}=    Execute Command
+    ...    dirname ${sos_report_tarfile}
+    ...    sudo=True    return_rc=True    return_stdout=True
+    Log    ${sos_report_dir}
+    Should Be Equal As Integers    0    ${rc}
+
+    ${rc}=    Execute Command
+    ...    tar xf ${sos_report_tarfile} -C ${sos_report_dir}
+    ...    sudo=True    return_rc=True    return_stdout=False
+    Should Be Equal As Integers    0    ${rc}
+
+    ${sos_report_untared}    ${rc}=    Execute Command
+    ...    find ${sos_report_dir} -type d -name "sosreport-*"
+    ...    sudo=True    return_rc=True    return_stdout=True
+    Should Be Equal As Integers    0    ${rc}
```

Comment:
We can simplify this code by using `Command Should Work` from `microshift-host.resource`

Reply:
thanks, replaced in all `Execute Command` keywords